### DISCUSSION TITLE
not being able to use osu!lazer previous songs or skins when resetting pc.

### DISCUSSION THREAD

**Discussion by gameplanetworld** [ppy/osu#29864] [Q&A] [answered]
osu!stable is all run within 1 folder. the executable, songs, etc. are all stored in 1 folder.
meanwhile in osu!lazer is stored in 2 seperate folders.

i installed osu!lazer and changed the folder destination to my D:/ drive. but osu!lazer has appdata stuff (the executable) that was still stored on my C:/ drive. meaning when i reset my pc my C:/ drive reset but my D:/ was fine but now i was not able to launch osu!lazer anymore because the executable was deleted.
in osu!stable even if i reset my pc the osu! folder was not touched so i could still play and everyting.

even when i reinstalled osu!lazer i couldn't use my old osu!lazer songs or skins. i tried copying the files and even importing using the ingame startup popup but nothing worked.
would be nice if osu!lazer had a copy of the appdata inside the folder that has songs stored aswel. making it so that if anything gets deleted outside of the osu!lazer folder it would be able to be run.

sorry if this is not the way to report it im just trying to help some way.

**Comment by cihe13375** [comment]
> even when i reinstalled osu!lazer i couldn't use my old osu!lazer songs or skins

After reinstallation, have you tried settings - change folder location - select the lazer folder in your D: drive?

**Reply by gameplanetworld** [reply]
yes i have tried but did not work

**Comment by SupDos** [comment]
In your case all you needed to do was reinstall lazer, go into settings, click "choose folder location" then pick the folder on your D drive that you had previously. Lazer will detect there is data in that folder and switch to using it

**Comment by gameplanetworld** [comment, marked as answer]
go to setting > change folder location > choose the previous lazer folder.
for some reason it sometimes works and sometimes it doesn't work.
could be because i had the osu-lazer folder in a folder called the same name causeing the code to fuck up and not see data.
